foo.rb

```diff
@@ -1,6 +1,5 @@
 class Block
   def foo
-    puts "one"
-    yield
```

Comment:
@bion The above two (removed) lines work as the body of the `foo` method if I try it in PRY, but I'm having trouble getting my test to pass.  Should I even be testing against STOUT?